README.md

```diff
@@ -123,6 +123,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: [Join the Discord](https://discord.gg/python), post on an unclaimed Issue with your interest and mention your in-server name.
 - **Notes**:  See this page for their [Reddit announcement](https://www.reddit.com/r/Python/comments/dbj0vh/python_discord_hacktoberfest_2019/). [Contributing Guides](https://pythondiscord.com/pages/contributing/) are available on their website.
 
+### Q
+
+### **QMK**
+
+- **Swag**: Limited edition QMK T-Shirt, Keyboard for first-time contributors
+- **Requirements**: Interact 4 times with QMK repositories.
+- **How to sign up**: No formal sign up announced.
```

Comment:
The signup link is at the bottom of their website and links to [here](https://docs.google.com/forms/d/e/1FAIpQLSeFt0mmW2e8tFgTsGGlSUET_8dpGiyuOTZKyS5g2jcIM20d-Q/viewform)